wrfhydropy/core/model.py

```diff
@@ -123,15 +148,15 @@ def __init__(self,
         self.wrf_hydro_exe = None
         """pathlib.Path: pathlib.Path to wrf_hydro.exe file generated at compile-time."""
 
-
         # Set attributes
-        ## Get code version
+        # Get code version
         with self.source_dir.joinpath('.version').open() as f:
             self.version = f.read()
 
-        ## Load compile options
-        compile_json = json.load(self.source_dir.joinpath('compile_options.json').open())
-        if 'nwm' in self.model_config:
+        # Load compile options
```

Comment:
There seems to be a lot of default behavior here. We should probably include a description in the docstring for those config files about the behavior

Reply:
Let's do a "base" for compile time options.

Reply:
"base" usage is implement in a backwards-compatible way. It passes the tests. But we probably want to test it more in the real world....